src/renderer/components/Wallet/WalletDetails/WalletDetails.vue

```diff
@@ -38,6 +38,13 @@
             </strong>
           </I18n>
           <I18n
+            tag="span"
+            class="font-semibold pl-6 border-r border-theme-line-separator"
+            path="WALLET_DELEGATES.RANK_BANNER"
+          >
+            <strong place="rank">{{ votedDelegate.rank }}</strong>
+          </i18n>
+          <i18n
```

Comment:
Please match the case of the above 2 lines (capital first letter)

Reply:
Done.